test/integration/targets/parsing/no_extras.yml

```
@@ -0,0 +1,7 @@
+- hosts: all
+  gather_facts: false
+  tasks:
+  - name: ensure bad extra vars parameter does not create bad var
+    assert:
+      that:
+        - _raw_params is not defined
```

Include an assertion for the `-e` value which is parsed:

```suggestion
        - _raw_params is not defined
        - valid == "as string"
```